alacritty_terminal/src/display.rs

```diff
@@ -521,14 +521,16 @@ impl Display {
                     // Iterate over all non-empty cells in the grid
                     for cell in grid_cells {
                         // Update underline/strikeout
-                        rects.update_lines(&cell, &size_info, &metrics);
+                        lines.update_lines(&cell);
```

Comment:
`lines.update` might be better for naming.

Reply:
Sure, just mechanically renamed this.